### ISSUE TITLE
Moloch molochpcapread couldn't stat oui file

### ISSUE THREAD

**Issue by b4b857f6ee** [StamusNetworks/SELKS#347]
Hello,

Fresh install
first setup and upgrade.
Fix a nginx problem with kibana with some issue here, just modify the nginx configuration :).

I configure option 2 FPC.
I don't have internet on the system.

And the molochpcapread didn't not work, i guess because about the no internet access?
Any idea :)?

![image](https://user-images.githubusercontent.com/36012515/140313546-eb6172c8-1cec-4b32-b23e-093f0447f3b0.png)

**Comment by b4b857f6ee** [comment]
![image](https://user-images.githubusercontent.com/36012515/140315124-98f2c211-b9a3-4aad-a90c-18116d3febd9.png)

**Comment by pevma** [comment]
It is opening , so it seems there is just no data?

**Comment by b4b857f6ee** [comment]
> 
> 
> It is opening , so it seems there is just no data?

Yes and the service is not started... i don't understand why....
I got data on the scirius, on evebox, on kibana... but why not in moloch.... :/

**Comment by pevma** [comment]
The Moloch service starts with a delay on purpose as it depends on ES. You can always restart it  -  fyi - https://github.com/StamusNetworks/SELKS/wiki/Services-status-check 
Otherwise look for err message in the log file  (the service shows it)

**Comment by b4b857f6ee** [comment]
> 
> 
> The Moloch service starts with a delay on purpose as it depends on ES. You can always restart it - fyi - https://github.com/StamusNetworks/SELKS/wiki/Services-status-check Otherwise look for err message in the log file (the service shows it)

i already show it here 
![image](https://user-images.githubusercontent.com/36012515/140373728-edc51234-0dd9-484d-aa73-9d97629245ab.png)

**Comment by b4b857f6ee** [comment]
![image](https://user-images.githubusercontent.com/36012515/140373936-4bee4dfe-a564-4da2-a8f2-1501ff5495ea.png)


And the log

![image](https://user-images.githubusercontent.com/36012515/140374024-20634381-6b74-4248-b526-502158ac3f36.png)

**Comment by pevma** [comment]
I think the first time set up maybe did not finish properly, I think it be due to the fact you had no internet connection for the setup ?

**Comment by b4b857f6ee** [comment]
> 
> 
> I think the first time set up maybe did not finish properly, I think it be due to the fact you had no internet connection for the setup ?

I was having it, but i done it one more time without it yes.
I really need it?

**Comment by b4b857f6ee** [comment]
> 
> 
> I think the first time set up maybe did not finish properly, I think it be due to the fact you had no internet connection for the setup ?

I've just done it with internet. Not error. I got all running this time. I got pcacp in the /data/moloch/raw but still, nothing on the moloch interface x'D

in the /data/moloch/etc/config.ini i got this, so i guess the starting script didn't change the listening interface i have choose?
![image](https://user-images.githubusercontent.com/36012515/140503542-864ec8ed-be56-4cfd-b2f5-ab87c41da89a.png)

**Comment by b4b857f6ee** [comment]
I change it with my suricata interface. I got 5 packet on moloch and.... nothing more ^^.
Even if the pcap grow 
![image](https://user-images.githubusercontent.com/36012515/140505938-df5ac4c7-b22f-4d61-b7e0-6abd3e152cca.png)

**Comment by b4b857f6ee** [comment]
Ok i restart, i can see all but i can't see the last. nothing after 12h50, ant it's now 13h02. Strange no ?
![image](https://user-images.githubusercontent.com/36012515/140507449-76fafc43-d960-4c89-9366-7d7058a8f5f2.png)

**Comment by b4b857f6ee** [comment]
I got this in capture.log

![image](https://user-images.githubusercontent.com/36012515/140510425-fa56dca9-7a59-4396-a76a-d9d5ad4fb939.png)

**Comment by b4b857f6ee** [comment]
Ok i think i got a delta of 30 min before having the log in Moloch, normal?

**Comment by nour1509** [comment]
Hi i need your help. i have the same problem ! but i have setup the arkime using internet but always the same 
problem

**Comment by pevma** [comment]
Did you try selecting different time span ?
